src/ceph-volume/ceph_volume/configuration.py

```diff
@@ -59,9 +62,16 @@ def load(abspath=None):
         raise RuntimeError('Unable to read configuration file: %s' % abspath)
 
 
-class Conf(configparser.SafeConfigParser):
+if sys_version_info.major >= 3:
```

Comment:
This boiler plate if/else block is almost identical to the one at the top. Can you move the `conf_parentclass` definition there instead to avoid having this repetition?

Reply:
Nice!